db/queries/statuses/queries.py

```diff
@@ -243,3 +245,10 @@ def update_statuses(application_id: str, form_name: str, is_summary_page_submitt
 
     update_application_status(application, round.feedback_survey_config)
     db.session.commit()
+    if isinstance(round.deadline, datetime):
```

Comment:
It doesn't feel like this belongs in this function - maybe have a separate one that determines round status.

Reply:
Fixed